[2023-10-02T03:42:17.121+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Valdi_Pipeline.get_data_from_postgresql scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2023-10-02T03:42:17.140+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Valdi_Pipeline.get_data_from_postgresql scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2023-10-02T03:42:17.142+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2023-10-02T03:42:17.143+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2023-10-02T03:42:17.144+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2023-10-02T03:42:17.171+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): get_data_from_postgresql> on 2023-10-01 00:00:00+00:00
[2023-10-02T03:42:17.183+0000] {standard_task_runner.py:52} INFO - Started process 202 to run task
[2023-10-02T03:42:17.188+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Valdi_Pipeline', 'get_data_from_postgresql', 'scheduled__2023-10-01T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_Rivaldi_Valensia_DAG.py', '--cfg-path', '/tmp/tmpzj94vebu', '--error-file', '/tmp/tmpnq2ai82k']
[2023-10-02T03:42:17.194+0000] {standard_task_runner.py:80} INFO - Job 13: Subtask get_data_from_postgresql
[2023-10-02T03:42:17.343+0000] {task_command.py:371} INFO - Running <TaskInstance: Valdi_Pipeline.get_data_from_postgresql scheduled__2023-10-01T00:00:00+00:00 [running]> on host 5f5d0b98abe4
[2023-10-02T03:42:17.487+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Valdi
AIRFLOW_CTX_DAG_ID=Valdi_Pipeline
AIRFLOW_CTX_TASK_ID=get_data_from_postgresql
AIRFLOW_CTX_EXECUTION_DATE=2023-10-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-01T00:00:00+00:00
[2023-10-02T03:42:17.920+0000] {python.py:173} INFO - Done. Returned value was: None
[2023-10-02T03:42:17.945+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=Valdi_Pipeline, task_id=get_data_from_postgresql, execution_date=20231001T000000, start_date=20231002T034217, end_date=20231002T034217
[2023-10-02T03:42:18.008+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-02T03:42:18.109+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
